### PR TITLE
first header added

### PR DESCRIPTION
**Pull request by ekoyn**
How's the name of this header?

### CODE REVIEW

**Review by DRother21 — COMMENTED** [review]
Looks good to me!

**Review by DRother21 — DISMISSED** [review]
Changes?

**Review by DRother21 — COMMENTED** [review]
Review

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-
+<h1>This is a header</h1>
```
Comment (DRother21) [inline index.html]: Review

Comment (DRother21) [inline index.html]: Review

Comment (ekoyn) [inline index.html]: Starting a new review

**Review by DRother21 — APPROVED** [review]
Here is a review

File: index.html (hunk 1 above)
Comment (ekoyn) [reply index.html]: Ah yes, I'll make some edits

**Review by DRother21 — CHANGES_REQUESTED** [review]
Change Request

**Review by DRother21 — APPROVED** [review]

**Pull request merged**